### PR TITLE
Enforce creation limits: names > 2 letters, convos 2-6 ppl

### PR DESCRIPTION
**Pull request by psybuzz**
This PR enforces some creation limits.  It checks that names of new users are at least 2 letters long and no more than 128 letters.  It also ensures that conversations are created with only 2-6 people.

### CODE REVIEW

File: routes/conversation.js [hunk 1]
```diff
@@ -32,6 +32,11 @@ exports.create = function (req, res){
 		var people = idOrEmails.filter(function(idOrEmail){return idOrEmail.indexOf('@') === -1});
 		var emails = idOrEmails.filter(function(idOrEmail){return idOrEmail.indexOf('@') !== -1});
 
+		// Limit from 2 - 6 people, including oneself.
+		if (idOrEmails.length < 1 || idOrEmails.length > 5){
```
Comment (tonyxj) [inline routes/conversation.js]: Good point to double check guys!

File: routes/conversation.js [hunk 2]
```diff
@@ -32,6 +32,11 @@ exports.create = function (req, res){
...
+			resError(res, "Keep it intimate!  Please have 2 - 6 people in a conversation.");
```
Comment (tonyxj) [inline routes/conversation.js]: I'm nitpicking, but I'd swap out the "-" in "2 - 6 people" with a to:
"2 to 6"
So that it's more human readable.

Comment (psybuzz) [inline routes/conversation.js]: Sounds great Tony, 'to' is more clear.

**Comment by psybuzz** [comment]
Alright, I should have addressed the "2 to 6" and the self-exclusion comments in the last 2 commits.  Let me know if it's merge ready.

**Comment by tonyxj** [comment]
Awesome! LTGM.

**Pull request merged**